vaadin-grid-pro-flow-integration-tests/src/test/java/com/vaadin/flow/component/gridpro/test/BasicIT.java

```diff
@@ -26,6 +26,24 @@ public void editColumnsAdded() {
         Assert.assertEquals(columns.size(), 3);
     }
 
+    @Test
+    public void headerSorterIsRendered() {
+        GridTHTDElement headerCell = grid.getHeaderCell(1);
+        Assert.assertTrue(headerCell.$("vaadin-grid-sorter").exists());
+    }
+
+    @Test
+    public void headerSorterCanBeToggled() {
+        GridTHTDElement headerCell = grid.getHeaderCell(1);
+        GridTHTDElement bodyCell = grid.getCell(0, 1);
+
+        TestBenchElement sorter = headerCell.$("vaadin-grid-sorter").first();
+        Assert.assertEquals("Person 1", bodyCell.getInnerHTML());
+        sorter.click();
+        sorter.click();
+        Assert.assertEquals("Person 99", bodyCell.getInnerHTML());
+    }
+
```

Comment:
Add another test that verifies that sorting works properly after editing the names

Reply:
Added.